service/src/main/java/bio/terra/cbas/controllers/WiredRunSetsApiController.java

```diff
@@ -0,0 +1,29 @@
+package bio.terra.cbas.controllers;
+
+import bio.terra.cbas.config.CromwellServerConfiguration;
+import bio.terra.cbas.config.DatabaseConfiguration;
+import bio.terra.cbas.config.WdsServerConfiguration;
+import bio.terra.cbas.dao.*;
+import bio.terra.cbas.model.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class WiredRunSetsApiController extends RunSetsApiController {
+
+  @Autowired
+  public WiredRunSetsApiController(
+      CromwellServerConfiguration cromwellServerConfiguration,
+      WdsServerConfiguration wdsServerConfiguration,
+      DatabaseConfiguration databaseConfiguration) {
+
+    super(
+        wdsServerConfiguration.entitiesApi(),
```

Comment:
Ah, interesting. Right now it's not a problem because all these requests are in-app and so multi-user is moot... but as soon as that's not true, we'll run into problems